lambdas/navigator_config/index.py

```diff
@@ -16,7 +18,7 @@ def handler(event, context):
         config(props['DestBucket'], props['DestDir'], props['ConfigEsUrl'],
```

Comment:
this function is a bug waiting to happen. we should be passing in a dict instead of ten positional arguments that can be mixed up

Reply:
kwargs would also be OK

Reply:
will fix next week